src/languageservice/parser/custom-tag-provider.ts

```diff
@@ -0,0 +1,65 @@
+import { Tags, isSeq, isMap, YAMLMap, YAMLSeq } from 'yaml'
+import { filterInvalidCustomTags } from '../utils/arrUtils';
+
+class CommonTagImpl {
+  public tag: string;
+  public readonly type: string;
+  default: never;
```

Comment:
`public` is the default. If you left it off to make it private it will not work. It would be better to be consistent and use public on all public fields or none of them.